WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -138,49 +133,28 @@ final class BloggingPromptsServiceTests: CoreDataTestCase {
     }
 
     func test_fetchPrompts_givenNoParameters_assignsDefaultValue() throws {
-        let expectedDifferenceInDays = 10
         let expectedNumber = 25
 
         // call the fetch just to trigger default parameter assignment. the completion blocks can be ignored.
         service.fetchPrompts(success: { _ in }, failure: { _ in })
 
         // calculate the difference and ensure that the passed date is 10 days ago.
-        let date = try passedDate()
-        let differenceInDays = try XCTUnwrap(Self.calendar.dateComponents([.day], from: date, to: Date()).day)
-        XCTAssertEqual(differenceInDays, expectedDifferenceInDays)
+        let expectedDate = dateFormatter.string(from: service.defaultStartDate)
+        XCTAssertEqual(expectedDate, try passedDate())
 
         // ensure that the passed number parameter is correct.
         let numberParameter = try XCTUnwrap(passedNumber())
         XCTAssertEqual(numberParameter, expectedNumber)
     }
 
     func test_fetchPrompts_passesTheDateCorrectly() throws {
```

Comment:
This test only needs to assert that the given date matches the URL parameter – no need for any specific date math here